term_proposal/evan11401/project.md

```diff
@@ -0,0 +1,84 @@
+# High Dynamic Range Imaging using Multiple Exposure Images with Tone Mapping
+![img1](./img1.jpg)
+## Motivation
+The real world is high dynamic range, but the camera is limited dynamic range. In case to get the image that looks more like the real world image, high dynamic range imaging (HDRI) is invented.
+
+HDRI is widely used in the imaging and photography industry, such as the APPLE iPhone's camera also use this to get a better image.
+
+## Target
+I have read some tutorial to teach people how to implement the HDRI algorithm, but most of the function is provided by the OpenCV library.
+
+I know the library is powerful, but I think if I can do it on my own, I will learn the details and understand what it works. Moreover, I want to design the architecture like moving the time-consuming computation into c++, creating a user interface that users can easily use this program, and then merge it together.
+
+
+## Implement
+
+### 1. Upload several images with different exposures 
```

Comment:
It seems that you'd like to make this project a web / online service.  You can do that, but I suggest you leave it as an optional goal.  The scope of this course cannot give you score for the service part.  We focus on the system design and implementation.  The "usability" for numerical software is about the API, so it's not for "end users" (who don't code).

But a web UI is definitely good for a thing to be widely used.  It's a good thing.

Reply:
I already update the UI explanation, please check it.
Thanks for your advice.